circe-akka-serializer/src/main/resources/reference.conf

```diff
@@ -1,5 +1,19 @@
 # Default configuration
 
 org.virtuslab.ash {
-  verbose-debug-logging = off
+  circe {
+    verbose-debug-logging = off
```

Comment:
See below

Reply:
it showed up that this config is relatted only to circe akka serializer and thus it can stay as it is (under circe path)